Persephone/Controllers/AlbumDetailView.swift

```diff
@@ -47,16 +47,32 @@ class AlbumDetailView: NSViewController {
 
   func getAlbumSongs(for album: Album) {
     App.mpdClient.getAlbumSongs(for: album.mpdAlbum) { (mpdSongs: [MPDClient.MPDSong]) in
-      self.dataSource.albumTracks = mpdSongs.map {
-        return Song(mpdSong: $0)
-      }
+      self.dataSource.setAlbumSongs(
+        mpdSongs.map { Song(mpdSong: $0) }
+      )
+
+      self.getBigCoverArt(song: self.dataSource.albumSongs.first!.song ?? self.dataSource.albumSongs[1].song!)
```

Comment:
What about something like:

```swift
self.dataSource.albumSongs.compactMap { $0.song }.first
```

Reply:
I completely refactored this bit. I intended to before making the PR but forgot all about it.